src/Servers/IIS/IIS/test/testassets/InProcessNewShimWebSite/InProcessNewShimWebSite.csproj

```diff
@@ -63,13 +63,4 @@
     </PackageReference>
     <Reference Include="xunit.assert" />
   </ItemGroup>
-
-  <!-- Repeat Build target for win-x64 to allow later Publish w/ NoBuild=true. -->
-  <Target Name="BuildX64"
-      BeforeTargets="Build"
-      Condition=" '$(RuntimeIdentifier)' != 'win-x64' AND '$(TargetArchitecture)' != 'arm' ">
-    <MSBuild Projects="$(MSBuildProjectFullPath)"
-        Properties="RuntimeIdentifier=win-x64;ReferenceTestTasks=false"
```

Comment:
Suggest adding `BuildProjectReferences=false` to these `Properties` instead of removing the whole target. Same in the other project

Reply:
Why is this target necessary at all given that all the tests are passing?

Reply:
You're right, these two `<Target/>`s are now redundant and cause problems.

Took a bit to find the change that made these `<Target/>`s unnecessary. They were mandatory back when I did #39998 because the `Publish` target was only run in the Helix build step and that step uses `-noBuild`. `Publish` fails if the project hasn't yet been built (in this case, for the `$(RuntimeIdentifier)` of interest).

This changed when I did #40379 because [the `CopyAssets` target in `FunctionalTest.props`](https://github.com/dotnet/aspnetcore/blob/b24acedc162732423811c4fc3e02a3e6a6283681/src/Servers/IIS/IIS/test/FunctionalTest.props#L11) now runs in the prior build step, without `-noBuild`.